contracts/AnchorRepository.sol

```diff
@@ -24,46 +23,32 @@ contract AnchorRepository is Initializable {
 
   struct PreAnchor {
     bytes32 signingRoot;
-    uint48 centrifugeId;
+    address centrifugeId;
     uint32 expirationBlock;
   }
 
-  // address for the Identity Registry
-  address identityRegistry;
   // store precommits
   mapping(uint256 => PreAnchor) public preCommits;
   // store commits
   mapping(uint256 => bytes32) public commits;
   // The number of blocks for which a precommit is valid
   uint256 constant internal expirationLength = 15;
 
-  function initialize(
-    address _identityRegistry
-  )
-  public
-  initializer
-  {
-    identityRegistry = _identityRegistry;
-  }
-
   // @param _anchorId Id for an Anchor.
   // @param _signingRoot merkle tree for a document that does not contain the signatures
   // @param _centrifugeId Id for the Identity that wants to precommit
   // @param _signature Signed data
   function preCommit(
     uint256 _anchorId,
     bytes32 _signingRoot,
-    uint48 _centrifugeId,
+    address _centrifugeId,
```

Comment:
I thought we should never send the `centrifugeId`. Instead we should use `msg.sender` or is the idea to change it step by step?